app/models/site.rb

```diff
@@ -7,10 +7,48 @@ class Site < ActiveRecord::Base
   accepts_nested_attributes_for :account, update_only: true
 
   class << self
-    delegate :account, :application_name, :institution_name, :institution_name_full, :reload, :update, to: :instance
+    delegate :account, :application_name, :institution_name,
+             :institution_name_full, :reload, :update, :announcement_text,
+             :marketing_text, :featured_researcher, to: :instance
 
     def instance
       first || create
     end
   end
+
+  def announcement_text
```

Comment:
We're getting `ContentBlock` from sufia, I imagine? Instead of adding our own accessors here, would it make sense to set up the expected AR associations to that model?